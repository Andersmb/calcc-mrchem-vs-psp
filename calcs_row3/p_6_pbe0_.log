Starting job 5688420 on c1-[26,39],c2-32,c5-[34-36] at Sat May 21 09:57:20 CEST 2022

Copying p_6_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/triplets
Copying p_6_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/triplets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688420       p_6_pbe0_        180                                             00:25:15      0:0 
5688420.bat+      batch         36        1   00:00:02          0   00:00:02   00:25:15      0:0 
5688420.ext+     extern        180        6   00:00:00          0   00:00:00   00:25:15      0:0 
5688420.0      mrchem.x        180       30   00:24:58         29   01:06:37   00:25:04      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688420                                                                          
5688420.bat+     19340K          0     19340K        0              0          0 
5688420.ext+          0          0          0        0              0          0 
5688420.0      7806296K         29 3716372206        0              4          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688420                                                                                               
5688420.bat+       16.91M               0         16.91M        0.03M                0          0.03M 
5688420.ext+        0.00M               0          0.00M            0                0              0 
5688420.0           6.77M               6          6.73M       67.01M               17         44.01M 

Job 5688420 completed at Sat May 21 10:22:34 CEST 2022
