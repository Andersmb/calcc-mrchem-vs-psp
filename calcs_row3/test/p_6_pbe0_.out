                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -1.142100
    1       F            :        0.000000        0.000000        1.861997
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              13
 Beta electrons          :                                              11
 Total electrons         :                                              24
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999454589e-01
    1    1    a          :                              9.999999141334e-01
    2    1    a          :                              9.999998176834e-01
    3    1    a          :                              1.000000101950e+00
    4    1    a          :                              9.999999975075e-01
    5    1    a          :                              9.999999989170e-01
    6    1    a          :                              1.000000393224e+00
    7    1    a          :                              1.000008037822e+00
    8    1    a          :                              9.999941760902e-01
    9    1    a          :                              1.000000595387e+00
   10    1    a          :                              9.999954134302e-01
   11    1    a          :                              9.999992972043e-01
   12    1    a          :                              9.999992956020e-01
   13    1    b          :                              1.000000003736e+00
   14    1    b          :                              9.999999297130e-01
   15    1    b          :                              9.999998177410e-01
   16    1    b          :                              1.000000101947e+00
   17    1    b          :                              9.999999975454e-01
   18    1    b          :                              9.999999977291e-01
   19    1    b          :                              1.000000391429e+00
   20    1    b          :                              1.000008040579e+00
   21    1    b          :                              9.999941802817e-01
   22    1    b          :                              1.000000595851e+00
   23    1    b          :                              9.999954143783e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                436.950599390776
 E-N energy              :            (au)              -1134.834624972148
 Coulomb energy          :            (au)                247.752791181758
 Exchange energy         :            (au)                 -8.166224297038
 X-C energy              :            (au)                -25.261024964673
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 44.938622828940
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -483.558483661325
 Nuclear energy          :            (au)                 44.938622828940
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.386198608324e+02
                         :      (kcal/mol)             -2.752381159638e+05
                         :        (kJ/mol)             -1.151596277940e+06
                         :            (eV)             -1.193545392369e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.780896615643
    1    1    a          :            (au)                -24.574229812925
    2    1    a          :            (au)                 -6.652062431114
    3    1    a          :            (au)                 -4.791063139350
    4    1    a          :            (au)                 -4.790949891441
    5    1    a          :            (au)                 -4.790355480223
    6    1    a          :            (au)                 -1.225566928205
    7    1    a          :            (au)                 -0.625868952578
    8    1    a          :            (au)                 -0.449151552001
    9    1    a          :            (au)                 -0.448835865332
   10    1    a          :            (au)                 -0.428974114248
   11    1    a          :            (au)                 -0.239274510357
   12    1    a          :            (au)                 -0.239108501099
   13    1    b          :            (au)                -76.773050029901
   14    1    b          :            (au)                -24.565615111994
   15    1    b          :            (au)                 -6.643165987512
   16    1    b          :            (au)                 -4.785847386024
   17    1    b          :            (au)                 -4.773475617851
   18    1    b          :            (au)                 -4.773462924004
   19    1    b          :            (au)                 -1.203396000660
   20    1    b          :            (au)                 -0.571370656636
   21    1    b          :            (au)                 -0.403788303289
   22    1    b          :            (au)                 -0.403485154841
   23    1    b          :            (au)                 -0.390519260776
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -251.323514228002
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.898979e+00        -438.619860832385         -4.386199e+02
    1          2.499028e-01        -440.901935775597         -2.282075e+00
    2          1.004082e-01        -440.980830678972         -7.889490e-02
    3          6.198795e-02        -440.994658869204         -1.382819e-02
    4          3.847442e-02        -440.998880880466         -4.222011e-03
    5          5.590543e-02        -441.002204414473         -3.323534e-03
    6          1.582774e-02        -441.002685868499         -4.814540e-04
    7          1.646658e-02        -441.003078347726         -3.924792e-04
    8          6.384209e-03        -441.003131328089         -5.298036e-05
    9          5.840951e-03        -441.003161298750         -2.997066e-05
   10          2.244422e-03        -441.003170077650         -8.778900e-06
   11          2.271674e-03        -441.003176062481         -5.984831e-06
   12          1.078107e-03        -441.003177432974         -1.370493e-06
   13          7.551796e-04        -441.003178038854         -6.058794e-07
   14          9.014849e-04        -441.003178609311         -5.704574e-07
   15          4.960382e-04        -441.003178835060         -2.257484e-07
   16          2.634680e-04        -441.003178919451         -8.439110e-08
   17          1.507942e-04        -441.003178950458         -3.100763e-08
   18          1.321355e-04        -441.003178974561         -2.410258e-08
   19          9.711365e-05        -441.003178983262         -8.700681e-09
---------------------------------------------------------------------------
                      SCF converged in 19 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -1.142100
    1       F            :        0.000000        0.000000        1.861997
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                440.259553642587
 E-N energy              :            (au)              -1140.676990436003
 Coulomb energy          :            (au)                247.941206616412
 Exchange energy         :            (au)                 -8.171421944474
 X-C energy              :            (au)                -25.294149690723
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 44.938622828940
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -485.941801812202
 Nuclear energy          :            (au)                 44.938622828940
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.410031789833e+02
                         :      (kcal/mol)             -2.767336706711e+05
                         :        (kJ/mol)             -1.157853678839e+06
                         :            (eV)             -1.200030731159e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.282992981573
    1    1    a          :            (au)                -24.805518572386
    2    1    a          :            (au)                 -6.675770681540
    3    1    a          :            (au)                 -4.813722107406
    4    1    a          :            (au)                 -4.813722079899
    5    1    a          :            (au)                 -4.810156503948
    6    1    a          :            (au)                 -1.270378062030
    7    1    a          :            (au)                 -0.659848839163
    8    1    a          :            (au)                 -0.508395983590
    9    1    a          :            (au)                 -0.508395964811
   10    1    a          :            (au)                 -0.473739141858
   11    1    a          :            (au)                 -0.255304522502
   12    1    a          :            (au)                 -0.255304520980
   13    1    b          :            (au)                -77.274133817830
   14    1    b          :            (au)                -24.799345335806
   15    1    b          :            (au)                 -6.666664291383
   16    1    b          :            (au)                 -4.805747872073
   17    1    b          :            (au)                 -4.794757500784
   18    1    b          :            (au)                 -4.794757497446
   19    1    b          :            (au)                 -1.257346967945
   20    1    b          :            (au)                 -0.617937964405
   21    1    b          :            (au)                 -0.483308366908
   22    1    b          :            (au)                 -0.483308335247
   23    1    b          :            (au)                 -0.419180283870
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -253.529738195385
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000001        0.000007        0.090122
 Magnitude               :            (au)                        0.090122
                         :         (Debye)                        0.229068
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.373526
 Magnitude               :            (au)                        0.373526
                         :         (Debye)                        0.949409
---------------------------------------------------------------------------
 Total vector            :        0.000001        0.000007       -0.283404
 Magnitude               :            (au)                        0.283404
                         :         (Debye)                        0.720341
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 25m  0s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
